tests/spread/plugins/autotools/snaps/autotools-hello/snap/snapcraft.yaml

```diff
@@ -18,4 +18,4 @@ build-packages: [gcc]
 parts:
   make-project:
     plugin: autotools
-    source: http://ftp.gnu.org/gnu/hello/hello-2.10.tar.gz
+    source: hello-2.10.tar.gz
```

Comment:
why not unpack the tar here and use `source: .`?
this would make for a bit more of a pleasant deb.

Reply:
Because each one of these cause about a million static test failures, and our ignore list will balloon. Thoughts?

Reply:
fine then, if we get push back we can try and do it